common/src/main/java/discord4j/common/store/legacy/StateHolder.java

```diff
@@ -46,13 +51,18 @@
  * <li>Voice state store: {@code long} pair keys and {@link VoiceStateData} values.</li>
  * </ul>
  */
+@SuppressWarnings("rawtypes")
 public final class StateHolder {
 
     private static final Logger log = Loggers.getLogger(StateHolder.class);
 
     private final StoreService storeService;
     private final LongObjStore<ChannelData> channelStore;
     private final LongObjStore<GuildData> guildStore;
+    private final Store<LongLongTuple2, GuildScheduledEventData> guildEventsStore;
+
+    private final Store<LongLongTuple2, Set> guildEventsUsersStore;
```

Comment:
I understand a raw type is chosen because our legacy stores library does not support parameterized types

This means, in order to not break support with store backends that use JSON serialization (like redis), we need to use Long items instead of the `Id` class: instead of storing `Id` we'd store `Id#asLong`

That way the actual element type is preserved after deserialization.

Reply:
I updated the methods using this store to use only long boxed objects instead of Id.